python/lsst/source/injection/inject_coadd.py

```diff
@@ -63,7 +68,58 @@ class CoaddInjectConfig(  # type: ignore [call-arg]
 ):
     """Coadd-level configuration for source injection tasks."""
 
-    pass
+    n_fits_1 = Field[int](
+        doc="Perform this many RANSAC fits in the first round, to get a sample "
+        "of different slopes based on the different random samples of points used "
+        "in the fit.",
+        default=20,
+    )
+    n_fits_2 = Field[int](
+        doc="Perform this many RANSAC fits in the second round, to get a sample "
+        "of different slopes based on the different random samples of points used "
+        "in the fit.",
+        default=20,
+    )
+    threshold_scale_1 = Field[float](
+        doc="An outlier in the first RANSAC fit is farther from the fit line, "
+        "in terms of squared error, than this multiple of the initial linear MSE.",
+        default=0.1,
+    )
+    threshold_scale_2 = Field[float](
+        doc="An outlier in the second RANSAC fit is farther from the fit line, "
+        "in terms of squared error, than this multiple of the initial linear MSE.",
+        default=0.1,
+    )
+    max_trials_1 = Field[int](
+        doc="Maximum number of trials the first RANSAC fit is allowed to run.",
+        default=1000,
+    )
+    max_trials_2 = Field[int](
+        doc="Maximum number of trials the second RANSAC fit is allowed to run.",
+        default=1000,
+    )
+    variance_fit_seed_1 = Field[int](doc="Seed for first RANSAC fit of flux vs. variance.", default=0)
+    variance_fit_seed_2 = Field[int](doc="Seed for second RANSAC fit of flux vs. variance.", default=0)
```

Comment:
These would probably be cleaner (and others below) if the defaults were on a new line, in keeping with most other configs.